### ISSUE TITLE
Promote task-list accessibility workshop

### ISSUE THREAD

**Issue by ImranH-GDS** [alphagov/govuk-design-system#2620]
## What
Spread the word about the task-list accessibility workshop
## Why
So that people attend
## Who needs to work on this
Community designer
## Who needs to review this
Accessibility specialist
## Done when

- [x] Create Eventbrite
- [x] Post on Slack spaces
- [x] Create MailChimp
- [x] Promote on departmental mailing list
- [x] Promote on Twitter/Mastodon
